google/auth/external_account.py

```diff
@@ -125,6 +130,7 @@ def __init__(
         self._scopes = scopes
         self._default_scopes = default_scopes
         self._workforce_pool_user_project = workforce_pool_user_project
+        self._universe_domain = universe_domain or _DEFAULT_UNIVERSE_DOMAIN
```

Comment:
I don't think you need the `or` portion of this statement, since `universe_domain` uses `_DEFAULT_UNIVERSE_DOMAIN` as a default value

Reply:
It's a protection of user explicitly set `None` or empty string. I asked @arithmetic1728 about the implementation of 
https://github.com/googleapis/google-auth-library-python/blob/main/google/oauth2/service_account.py#L182-L185
is the same purpose.

Reply:
Hmm. Wouldn't throwing an invalid argument exception be better in that case? That seems like something users should be alerted to, rather then falling back to the default universe.

Reply:
Maybe keep a consistency to `service_account.Credentials`? Or we can consider to change that part together as well.

Reply:
Let's let @arithmetic1728 chime in, but the change can be deferred to a separate PR.

Reply:
When users provide None or empty string I think their intent is to use the default value so this treatment looks fine to me.  But if everyone wants to throw an exception, I don't mind changing it as well for consistency.